ExchangeSharp/API/Exchanges/Bl3p/ExchangeBl3pAPI.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+
+namespace ExchangeSharp.API.Exchanges.Bl3p
```

Comment:
This can be in the main ExchangeSharp namespace

Reply:
Should only the Exchange class be on the main namespace? Or is this also true for Exchange-specific models and other helper classes?

Reply:
Fixed in 84e1a15

Reply:
Just the ExchangeAPI class. The supporting exchange specific models and other helpers can be in their own namespace. But that can be a simplified one, such as [ExchangeSharp.Coinbase](https://github.com/jjxtra/ExchangeSharp/blob/master/ExchangeSharp/API/Exchanges/Coinbase/Models/CoinbaseTrade.cs).